docs/pages/guides/analyzing-bundles.mdx

```diff
@@ -10,77 +10,39 @@ Bundle performance varies for different platforms. For example, web browsers don
 
 ## Analyzing bundle size
 
-The libraries used in a project influence the size of the production JavaScript bundle. You can use [source map explorer](https://www.npmjs.com/package/source-map-explorer) to visualize the production bundle and identify which libraries are contributing to the bundle size.
+The libraries used in a project influence the size of the production JavaScript bundle. Starting from **Expo SDK 51**, you can use [Expo Atlas](https://github.com/expo/expo-atlas#readme) to visualize the production bundle and identify which libraries contribute to the bundle size.
```

Comment:
Good point, we could keep the source map analyzer below with a warning that it will eventually be deprecated (or at least add a warning that it's for SDK 50 and below)

Reply:
We can also use a `Collapsible` for SDK 50 instructions and I agree about adding warning/deprecation callout.

Reply:
Done

Reply:
I think the `Collapsible` might be a bit much, since the old docs also contains step components etc. So I added the docs back, but used the titles:
- Analyzing bundle size with Atlas
- Analyzing bundle size with source-map-explorer

Reply:
Thanks, Cedric!